projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -380,7 +381,11 @@ GET /organizers/?q=imagesCount:[5 TO 10]
 
 ### labels
 
+<<<<<<< HEAD
+Search for documents that have a specific label with the `labels` field.
+=======
 Search for documents that have a specific label (or hidden label) with the `labels` parameter.
+>>>>>>> b52c705c8e68707cb8a713bc93287b86f9a2e92a
```

Comment:
There are some artifacts from a merge conflict left over here. I think the sentence should be:

```
Search for documents that have a specific label (or hidden label) with the `labels` field.
```

(So a mix of the two)